src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/AddUsingsCodeActionParams.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions
+{
+    internal sealed class AddUsingsCodeActionParams
+    {
+        public Uri Uri { get; set; }
+        public string[] Namespaces { get; set; }
```

Comment:
How about `IReadOnlyCollection`?

Reply:
Why multiple namesapces? Maybe I'm misunderstanding what this part of the parameter represents. Is it the users `@namespace` directives? Is it the current set of `@using`'s in scope? The `using` namespace to add?